On ARM64.

>   In file included from /home/runner/work/PHEM/PHEM/app/src/main/jni/SrcShared/Hardware/EmCPU68K.cpp:14:0:
  /home/runner/work/PHEM/PHEM/app/src/main/jni/SrcShared/Hardware/EmCPU68K.cpp: In member function 'void EmCPU68K::ProcessException(ExceptionNumber)':
  /home/runner/work/PHEM/PHEM/app/src/main/jni/SrcShared/EmCommon.h:62:61: error: size of array '_dummy' is negative
   #define COMPILE_TIME_ASSERT(x) do { char _dummy[(x) ? 1 : -1]; UNUSED_PARAM (_dummy)} while (0)
                                                               ^
  /home/runner/work/PHEM/PHEM/app/src/main/jni/SrcShared/Hardware/EmCPU68K.cpp:1196:3: note: in expansion of macro 'COMPILE_TIME_ASSERT'
     COMPILE_TIME_ASSERT (sizeof (ExceptionStackFrame2) == 14);
     ^
  /home/runner/work/PHEM/PHEM/app/src/main/jni/SrcShared/EmCommon.h:62:61: error: size of array '_dummy' is negative
   #define COMPILE_TIME_ASSERT(x) do { char _dummy[(x) ? 1 : -1]; UNUSED_PARAM (_dummy)} while (0)
                                                               ^
  /home/runner/work/PHEM/PHEM/app/src/main/jni/SrcShared/Hardware/EmCPU68K.cpp:1212:3: note: in expansion of macro 'COMPILE_TIME_ASSERT'
     COMPILE_TIME_ASSERT (sizeof (ExceptionStackFrame1) == 6);
     ^

It looks like alignment issue. But `#pragma pack(2)` is written in [PalmPack.h](https://github.com/Florin9doi/PHEM/blob/master/app/src/main/jni/SrcShared/PalmPack.h), so I don't know why this happened (related [stackoverflow](https://stackoverflow.com/questions/54257525/)).

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -1,0 +1,61 @@
+name: Builder
+
+on:
+  push:
+    tags: '*'
+  workflow_dispatch:
+  
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Set up JDK 8
+      uses: actions/setup-java@v2
+      with:
+        java-version: '8'
+        distribution: 'adopt'
+    - name : install ndk
+      run: | 
+        sudo curl --silent --show-error --location --fail --retry 3 \
+          --output /tmp/$NDK_VERSION.zip \
+          https://dl.google.com/android/repository/$NDK_VERSION-linux-x86_64.zip
+
+        sudo echo "$NDK_SHA /tmp/$NDK_VERSION.zip" > /tmp/$NDK_VERSION.zip.sha1
+
+        sha1sum -c /tmp/$NDK_VERSION.zip.sha1
+
+        sudo rm -rf $ANDROID_SDK_ROOT/ndk-bundle
+        sudo unzip -q /tmp/$NDK_VERSION.zip -d $ANDROID_SDK_ROOT
+        sudo mv $ANDROID_SDK_ROOT/$NDK_VERSION $ANDROID_SDK_ROOT/ndk-bundle
+        sudo chown -R root:root $ANDROID_SDK_ROOT/ndk-bundle
+        sudo rm -f /tmp/$NDK_VERSION*
+
+        if [[ -d $ANDROID_SDK_ROOT/ndk-bundle && \
+          -n "$(ls -A $ANDROID_SDK_ROOT/ndk-bundle)" ]]; then
+          echo "Android NDK installed"
+        else
+          echo "Android NDK did not install successfully"
+          exit 1
+        fi
+      env:
+        NDK_SHA: 12cacc70c3fd2f40574015631c00f41fb8a39048
+        NDK_VERSION: android-ndk-r17c
+    - name: Validate Gradle wrapper
+      uses: gradle/wrapper-validation-action@e6e38bacfdf1a337459f332974bb2327a31aaf4b
+    - name: build
+      run: |
+        if [[ -n "${{ secrets.KEYSTORE_BASE64 }}" ]]; then
+          echo "${{ secrets.KEYSTORE_BASE64 }}" | base64 -d > release.keystore
+          export KEYSTORE_PASSWORD="${{ secrets.KEYSTORE_PASSWORD }}"
+          export KEY_ALIAS="${{ secrets.KEY_ALIAS }}"
+          export KEY_PASSWORD="${{ secrets.KEYSTORE_PASSWORD }}"
+        fi
+        chmod +x ./gradlew
+        ./gradlew assembleRelease
+    - uses: actions/upload-artifact@v2
+      with:
+        name: Package
+        path: app/build/outputs/apk

--- a/app/build.gradle
+++ b/app/build.gradle
@@ -11,21 +11,25 @@ android {
 
         versionCode 2190143
         versionName "1.43"
+        
+        flavorDimensions "architecture"
 
         productFlavors {
             arm {
                 versionNameSuffix "-arm"
+                dimension "architecture"
                 ndk {
                     abiFilters "armeabi-v7a"
                 }
             }
             intel {
                 versionNameSuffix "-intel"
+                dimension "architecture"
                 ndk {
                     abiFilters "x86"
                 }
             }
-        }
+       }
 
         externalNativeBuild {
             ndkBuild {
@@ -44,6 +48,9 @@ android {
         ndkBuild {
             path 'src/main/jni/Android.mk'
         }
+    }
+    lintOptions {
+      abortOnError false
     }
 }
 

